config/generator-theia/src/browser/browser-frontend-generator.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2017 TypeFox and others.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import { AbstractFrontendGenerator, FileSystem } from "../common";
+
+export class BrowserFrontendGenerator extends AbstractFrontendGenerator {
+
+    generate(fs: FileSystem): void {
+        this.doGenerate(fs, this.model.frontendModules);
+    }
+
+    protected compileIndexHtml(frontendModules: Map<string, string>): string {
+        return `<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
+  <script type="text/javascript" src="https://www.promisejs.org/polyfills/promise-6.1.0.js" charset="utf-8"></script>
```

Comment:
these is already an issue about it: https://github.com/theia-ide/theia/issues/251 someone should take care, should be pretty straightforward